README.md

```diff
@@ -164,6 +164,15 @@ first time.
 
 For example: `$BASE16_THEME_DEFAULT="solarized-light"` 
 
+### Default config path
+
+You can customize where the generated configuration of this script is 
+stored by setting the `$BASE16_CONFIG_PATH` environment variable before
+the `profile_helper` script is loaded. This variable defaults to
+`$HOME/.config/base16-project`.
+> If you are using oh-my-zsh you need to set this variable before 
```

Comment:
Why the `>`?

Reply:
I use these to nest notes in a document. This felt like an extra piece of information that was not necessary to include in the main body.

![CleanShot 2022-09-23 at 15 46 01](https://user-images.githubusercontent.com/4092035/191975105-8c183728-6b45-4f69-bf6f-9225100950e7.png)

I'm happy to change this though if you want me to!

Reply:
Ah ok. We've been using `>` as quotes so unless it's quoted text, it's not needed.

```suggestion

If you are using oh-my-zsh you need to set this variable before 
```